src/current/_includes/releases/v23.1/v23.1.24.md

```diff
@@ -0,0 +1,51 @@
+
+<h3 id="v23-1-24-bug-fixes">Bug fixes</h3>
+
+- Fixed handling in the [declarative schema changer]({% link v23.1/online-schema-changes.md %}#declarative-schema-changer) when columns are included in the `STORING()` clause of [`CREATE INDEX`]({% link v23.1/create-index.md %}). CockroachDB now checks if the column is virtual up-front, and properly detects when a column is already handled by an existing index when the column name has `UTF-8` characters. [#125208][#125208]
+- Fixed a bug where a change to a [user-defined type]({% link v23.1/create-type.md %}) could cause queries against tables using that type to fail with an error message like: `histogram.go:694: span must be fully contained in the bucket`. The change to the user-defined type could come directly from an [`ALTER TYPE`]({% link v23.1/alter-type.md %}) statement, or indirectly from an [ALTER DATABASE ... ADD REGION]({% link v23.1/alter-database.md %}#add-region) or [`DROP REGION`]({% link v23.1/alter-database.md %}#drop-region) statement (which implicitly change the `crdb_internal_region type`). This bug had existed since UDTs were introduced in v20.2. [#124855][#124855]
+- Fixed an issue where [adding a column]({% link v23.1/alter-table.md %}#add-column) with a default value of an empty array would not succeed. [#125328][#125328]
+- [`ALTER TABLE ... ADD CONSTRAINT UNIQUE`]({% link v23.1/alter-table.md %}#add-constraint) will now fail with a well-formed error message and code `42601` if a statement tries to add a unique constraint on an expression. [#125420][#125420]
+- Fixed a bug in v24.1, v23.2, and v23.1 where using `changefeed.aggregator.flush_jitter` with `min_checkpoint_frequency` set to zero could cause panics. [#125495][#125495]
+- Fixed a bug in which constant `LIKE` patterns containing certain sequences of backslashes did not become constrained scans. This bug has been present since v21.1.13 when support for building constrained scans from `LIKE` patterns containing backslashes was added. [#125537][#125537]
+- Fixed a bug that could cause calling a routine to return an unexpected `function ... does not exist` error. The bug is triggered when the routine is called twice using the exact same SQL query, and either: (a) the routine has polymorphic arguments, or: (b) in between the two calls, the routine is replaced by a routine with the same name and different parameters. This bug had existed since alpha versions of v23.1. [#123513][#123513]
+- Fixed the statistics estimation code in the [optimizer]({% link v23.1/cost-based-optimizer.md %}) so it does not use the empty histograms produced if histogram collection has been disabled during statistics collection due to excessive memory utilization. Now the optimizer will rely on distinct counts instead of the empty histograms and should produce better plans as a result. This bug had existed since CockroachDB v22.1. [#126158][#126158]
```

Comment:
nit
```suggestion
- Fixed the statistics estimation code in the [optimizer]({% link v23.1/cost-based-optimizer.md %}) so it does not use the empty histograms produced if histogram collection has been disabled during statistics collection due to excessive memory utilization. Now the optimizer will rely on distinct counts instead of the empty histograms and should produce better plans as a result. This bug had existed since v22.1. [#126158][#126158]
```